lua/tabnine.lua

```diff
@@ -9,6 +10,21 @@ local M = {}
 function M.setup(o)
 	config.set_config(o)
 
+	local v = vim.version()
+	if vim.version.lt(v, vim.version.parse(consts.min_nvim_version) or {}) then
```

Comment:
This is nice. Wasn't aware of this. We can remove the dependency in `semver.lua`